src/ert/_c_wrappers/job_queue/workflow_job.py

```diff
@@ -85,22 +129,42 @@ def fromFile(cls, config_file, name=None):
             if name is None:
                 name = os.path.basename(config_file)
 
-            content = _config_parser.parse(config_file)
-
-            def optional_get(key):
-                return content.getValue(key) if content.hasKey(key) else None
-
-            max_arg = optional_get("MAX_ARG")
-
-            return cls(
-                name,
-                optional_get("INTERNAL"),
-                optional_get("MIN_ARG"),
-                max_arg,
-                cls._make_arg_types_list(content, max_arg),
-                optional_get("EXECUTABLE"),
-                optional_get("SCRIPT"),
-            )
+            if not USE_NEW_PARSER_BY_DEFAULT:
+                old_content = _config_parser.parse(config_file)
+
+                def optional_get(key):
+                    return (
+                        old_content.getValue(key) if old_content.hasKey(key) else None
+                    )
+
+                max_arg = optional_get("MAX_ARG")
+
+                old_arg_types_list = cls._make_arg_types_list(old_content, max_arg)
+                return cls(
+                    name,
+                    optional_get("INTERNAL"),
+                    optional_get("MIN_ARG"),
+                    max_arg,
+                    old_arg_types_list,
+                    optional_get("EXECUTABLE"),
+                    optional_get("SCRIPT"),
+                )
+
+            else:
+                new_content_dict = new_workflow_job_parser(config_file)
+
+                new_types_list, new_max_argc = cls._make_arg_types_list_new(
+                    new_content_dict
+                )
+                return cls(
+                    name,
+                    new_content_dict.get("INTERNAL", None),
```

Comment:
flake8 wants you to use `new_content_dict.get("INTERNAL") and similarly for `MIN_ARGS`, `EXECUTABLE`, and `SCRIPT`